db/migrate/20211126142105_user_belongs_to_organisation.rb

```diff
@@ -0,0 +1,15 @@
+class UserBelongsToOrganisation < ActiveRecord::Migration[6.1]
+  def up
+    change_table :users, bulk: true do |t|
+      t.remove :organisation
+      t.belongs_to :organisation
```

Comment:
Could make this a foreign key by doing `t.belongs_to :organisation, foreign_key: true` but not done so for now since we might already have user records and it's not yet clear if this is necessary or even correct (must all users belong to an organisation?). For now I think using plain ActiveRecord association here is fine.